src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -6006,9 +5978,10 @@ namespace InternalHeatGains {
         } // ZoneITEq calc loop
 
         // Zone-level sensible heat index
-        for (Loop = 1; Loop <= NumOfZones; ++Loop) {
-            if (ZoneSumToutMinusTSup(Loop) != 0.0) {
-                ZnRpt(Loop).ITEqSHI = ZoneSumTinMinusTSup(Loop) / ZoneSumToutMinusTSup(Loop);
+        for (Loop = 1; Loop <= NumZoneITEqStatements; ++Loop) {
+            int ZN = ZoneITEq(Loop).ZonePtr;
+            if (ZnRpt(NZ).SumToutMinusTSup != 0.0) {
+                ZnRpt(ZN).ITEqSHI = ZnRpt(NZ).SumTinMinusTSup / ZnRpt(NZ).SumToutMinusTSup;
```

Comment:
The `ZN` and `NZ` is a bit icky, but not too bad since `ZN` is contained to this section.